src/Sulu/Bundle/PreviewBundle/Preview/Renderer/PreviewKernel.php

```diff
@@ -55,8 +60,12 @@ protected function getContainerClass()
         return $this->generateContainerClass(\get_parent_class());
     }
 
-    public function getRootDir()
+    public function getRootDir(/* $triggerDeprecation = true */)
     {
+        if (0 === \func_num_args() || \func_get_arg(0)) {
```

Comment:
Would you really build something like that in? Shouldn't we let that deprecation just happen?

Reply:
No the deprecation would then spam the deprecation log because of [internal calls](https://github.com/symfony/symfony/blob/92c4593e3cec1b806495512b1b8cd374728ae3d8/src/Symfony/Component/HttpKernel/Kernel.php#L93) of symfony in 4.4. Thats why they implemented this in the Symfony Kernel as we need the same behaviour I copied it from [there](https://github.com/symfony/symfony/blob/92c4593e3cec1b806495512b1b8cd374728ae3d8/src/Symfony/Component/HttpKernel/Kernel.php#L338-L340).  Because we sadly need to overwrite the getRootDir still for 4.4 version.